topic-operator/src/main/java/io/strimzi/operator/topic/LabelPredicate.java

```diff
@@ -136,6 +140,9 @@ public static LabelPredicate fromString(String string) throws IllegalArgumentExc
 
     @Override
     public boolean test(HasMetadata resource) {
+        if (labels == null || labels.size() == 0) {
```

Comment:
Do we need the null check? On a quick look it seems that this should probably not be null.